.github/workflows/label.yml

```diff
@@ -15,4 +13,3 @@ jobs:
       with:
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
         configuration-path: .github/label-config.yml
-        sync-labels: true
```

Comment:
According to the readme `sync-labels` does the following:

> Whether or not to remove labels when matching files are reverted or no longer changed by the PR

I think it should stay.

Reply:
The only file matching we do is on '**'. So the only way for a PR to remove all changes to the match would be to not change anything? Maybe I am misunderstanding this option, but it seems to me we do not need it.

Reply:
I think that just means it would be unlabeled if the pr stopped touching any files (which aren't in the `.github` or `.cargo` folders because of that bug)

That is, I think removing it is fine.

Reply:
Ah, the `.` directories are ignored? That would explain why this PR did not get the label...

Reply:
Yeah, there's an open issue in the actions repo (too much effort to link as I'm on mobile)

I debugged it to source before I found the issue lol - I found out about it because of the `S-Pre-Relicense` adding change

Reply:
@bjorn3 does this resolve your comment?